### ISSUE TITLE
Increase size of tap targets on mobile

### ISSUE THREAD

**Issue by collinbarrett** [collinbarrett/FilterLists#16]
The small + icons to expand list info on mobile are too small. The entire list row should be made "tap-able" to expand the list info.

**Comment by collinbarrett** [comment]
Won't fix on current site, will ensure this is done properly in new site being built.
